interface/modules/custom_modules/oe-module-weno/scripts/file_download.php

```diff
@@ -8,14 +8,13 @@
 use OpenEMR\Common\Crypto\CryptoGen;
 use OpenEMR\Common\Logging\EventAuditLogger;
 use OpenEMR\Modules\WenoModule\Services\PharmacyService;
-use OpenEMR\Modules\WenoModule\Services\TransmitProperties;
 use OpenEMR\Modules\WenoModule\Services\WenoLogService;
 
 $cryptoGen = new CryptoGen();
 $weno_username = $GLOBALS['weno_admin_username'] ?? '';
 $weno_password = $cryptoGen->decryptStandard($GLOBALS['weno_admin_password'] ?? '');
 $encryption_key = $cryptoGen->decryptStandard($GLOBALS['weno_encryption_key'] ?? '');
-$baseurl = "https://online.wenoexchange.com/en/EPCS/DownloadPharmacyDirectory";
+$baseurl = "https://dev.wenoexchange.com/en/EPCS/DownloadPharmacyDirectory";
```

Comment:
Do you mean to leave the URL as a dev location?  Or will this change in the future?  Might want to have an option to swap the system from dev to production mode.